src/utils/NucleicAcidSearchEngine.cpp

```diff
@@ -216,6 +219,9 @@ class NucleicAcidSearchEngine :
     registerStringOption_("fragment:mass_tolerance_unit", "<unit>", "ppm", "Unit of fragment mass tolerance", false, false);
     setValidStrings_("fragment:mass_tolerance_unit", ListUtils::create<String>("Da,ppm"));
 
+    registerIntOption_("fragment:min_charge", "<num>", -1, "Minimum precursor charge to be considered", false, false);
+    registerIntOption_("fragment:max_charge", "<num>", -999, "Maximum precursor charge to be considered (note: if this is higher than the precursor charge the precursor charge will be used", false, false);
```

Comment:
is it clear that higher charge == lower number? sounds a bit strange but maybe is ok

Reply:
I've clarified this in the doc now.